CONTRIBUTING.md

```diff
@@ -0,0 +1,180 @@
+# 👥 Contributing to the Kids First Dataservice API
+
+## Adding fields
+
+Over time new data comes into the Kids First ecosystem, required the
+Dataservice to adapt. In these cases, developers need to add new columns or
+change columns in the Dataservice DB and API.
+
+### Where to Add Fields
+
+In order to add a new field to the Dataservice, the developer needs to know
+which table to add the field to. All entity types in the Dataservice have a
+standard folder structure like this:
+
+dataservice/api/genomic_file
```

Comment:
This should be enclosed in a code block otherwise it doesn't format properly. So like 
```
fill tree here
```